storybook/src/elements/actions/actions.stories.js

```diff
@@ -0,0 +1,183 @@
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-add-to-collection-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-favorites-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-delete-blob-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-delete-document-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-download-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-export-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-link-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-lock-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-move-documents-down-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-move-documents-up-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-notifications-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-share-button.js';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-untrash-document-button.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-icons';
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { color, select, text, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import fakeServer from '../../data/nuxeo.data';
+import DocumentBuilder from '../../data/documents.data';
+import image from '../../img/nuxeo-elements-catalog.svg';
+import { listOfIcons } from '../../lists/icons';
+import '../../../.storybook/i18n';
+
+const documentBuilder = new DocumentBuilder()
+  .setFileContent('Nuxeo Logo', image)
+  .setPermissions(['Write', 'ManageWorkflows']);
+
+const DOCUMENTS = [documentBuilder.build(), documentBuilder.build(), documentBuilder.build()];
+
+const DOCUMENT_TRASHED = new DocumentBuilder()
```

Comment:
Since this document is only used in one story, it could be located there.
But, it is just a nitpick detail, since some IDE's would raise also a warning.